### PR TITLE
Fix regression with mobile toolbar

### PR DESCRIPTION
**Pull request by jasmussen**
This PR fixes two issues:

1. The block traversal button is hidden, it doesn't fit with translations.
2. Pinned extensions are hidden, they don't fit or even scale.

To expand on 1, we have in place a "clickthrough" method for blocks with nested contexts. One tap, for example, on a fully populated columns block selects the columns block. Another tap selects a column. A third tap, selects paragraphs inside, etc. To this argument, I think it's fair to hide the traversal button.

To expand on point 2, this was always the intent, and it's the reason every extension has to have a menu option also.

Before:

screen shot 2018-10-23 at 10 50 54

After:

screen shot 2018-10-23 at 11 24 10

screen shot 2018-10-23 at 11 24 16

### CODE REVIEW

File: packages/edit-post/src/components/header/header-toolbar/style.scss [hunk 1]
```diff
@@ -62,15 +72,3 @@
 		}
 	}
 }
-
-.edit-post-header-toolbar .table-of-contents {
-	display: none;
-
-	.is-sidebar-opened & {
```
Comment (jasmussen) [inline packages/edit-post/src/components/header/header-toolbar/style.scss]: This rule appeared to be obsolete. It made no visual change when I removed it.

**Review by youknowriad — APPROVED** [review]
LGTM 👍 

Do you mind adding quick changelog entries ;)

**Comment by jasmussen** [comment]
Updated changelog. Can you verify I did it right?

**Comment by youknowriad** [comment]
Yep looks good Thanks Joen

**Comment by youknowriad** [comment]
Oh wait, it should be `(Unreleased)`

**Comment by youknowriad** [comment]
Looks good

**Pull request merged**